fixups/RegLegacyFixups/InitializeFixup.cpp

```diff
@@ -255,12 +358,44 @@ void InitializeConfiguration()
 
                             specItem.remediationRecords.push_back(recordItem);
                         }
+                        else if (type.compare(L"Redirect") == 0)
```

Comment:
Update traceDataStream with this fixupConfig to capture telemetry for this fixup Type.

Reply:
Added